ghost/core/core/server/models/role.js

```
@@ -79,11 +80,11 @@ Role = ghostBookshelf.Model.extend({
 
         if (action === 'assign' && loadedPermissions.user) {
             let checkAgainst;
-            if (_.some(loadedPermissions.user.roles, {name: 'Owner'})) {
+            if (checkUserPermissionsForRole(loadedPermissions, 'Owner')) {
                 checkAgainst = ['Owner', 'Administrator', 'Editor', 'Author', 'Contributor'];
-            } else if (_.some(loadedPermissions.user.roles, {name: 'Administrator'})) {
+            } else if (checkUserPermissionsForRole(loadedPermissions, 'Administrator')) {
                 checkAgainst = ['Administrator', 'Editor', 'Author', 'Contributor'];
-            } else if (_.some(loadedPermissions.user.roles, {name: 'Editor'})) {
+            } else if (checkUserPermissionsForRole(loadedPermissions, 'Editor') || hasRole(loadedPermissions, 'Super Editor')) {
```

ESLint

[error] 'hasRole' is not defined.

(no-undef)